<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MtDatabaseAstClass</class>
 <widget class="QMainWindow" name="MtDatabaseAstClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MtDatabaseAst</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbOutput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>317</y>
      <width>790</width>
      <height>313</height>
     </rect>
    </property>
    <property name="title">
     <string>LOG</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="pteLOG"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gbInput">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>790</width>
      <height>314</height>
     </rect>
    </property>
    <property name="title">
     <string>DBAssistant</string>
    </property>
    <widget class="QPlainTextEdit" name="pteSQL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>511</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTableList">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>100</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数据表：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDbName">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExeSql">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>执行查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTest">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>测试接口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数据库类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>140</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>主机地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDbHost">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>140</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDbPort">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>180</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>主机端口：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登录用户：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leUser">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>220</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePassword">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>260</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登录密码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAbort">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>断开数据库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>270</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>连接数据库</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbDbType">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>SQLITE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ORACLE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MYSQL</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MtDatabaseAst.qrc"/>
 </resources>
 <connections/>
</ui>
